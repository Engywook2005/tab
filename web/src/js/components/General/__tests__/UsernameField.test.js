/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'

const getMockProps = () => ({
  usernameDuplicate: false,
  otherError: false,
})

describe('UsernameField tests', function() {
  it('renders without error', function() {
    const UsernameField = require('js/components/General/UsernameField').default
    const mockProps = getMockProps()
    shallow(<UsernameField {...mockProps} />)
  })
})
