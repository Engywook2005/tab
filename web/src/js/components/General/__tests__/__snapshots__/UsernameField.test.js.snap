// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsernameField tests matches expected snapshot 1`] = `
<TextField
  data-test-id="username-field-text-input"
  disabled={false}
  errorText={null}
  floatingLabelFixed={false}
  fullWidth={false}
  id="username-input"
  multiLine={false}
  rows={1}
  type="text"
  underlineShow={true}
/>
`;
